test/tasks-test.coffee

```diff
@@ -0,0 +1,87 @@
+SinonChai = require('sinon-chai')

```

Comment:
Should these be differentiated (as in tasks be in their own test folder and unit tests be in their own folder?)